webpack/webpack.prod.cjs

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { mergeWithRules } = require('webpack-merge');
+
+const generateConfig = require(__dirname + '/webpack.config.cjs');
```

Comment:
Why not just require with relative path?

Reply:
People would run `npm` commands at the root of the repo, so the relative path IS `./webpack/webpack.config.cjs`. For consistency, I'll just concat the strings here.

Reply:
a relative path is resolved from script file itself, not cwd... it's always `./webpack.config.cjs`

Reply:
Ah, I tried and it worked.
I didn't remember why I add `__dirname` now, I'll change like that, Thanks.